app.js

```diff
@@ -180,68 +131,50 @@ app.get('/heapdump', (req, res, next) =>
 )
 
 app.post('/shutdown', function(req, res) {
-  appIsOk = false
-  return res.send()
+  exceptionHandler.setNotOk()
+  res.sendStatus(200)
 })
 
 app.get('/status', function(req, res) {
-  if (appIsOk) {
-    return res.send('filestore sharelatex up')
+  if (exceptionHandler.appIsOk()) {
+    res.send('filestore sharelatex up')
   } else {
     logger.log('app is not ok - shutting down')
-    return res.send('server is being shut down', 500)
+    res.send('server is being shut down').status(500)
```

Comment:
```suggestion
    res.status(500).send('server is being shut down')
```

`.send` is adding a default status code of 200 and calling `.end` with the payload. In order to set a different status code, we need to call `.status` first.